### ISSUE TITLE
Update the wording of the persistent notification

### ISSUE THREAD

**Issue by Claire1817** [element-hq/element-android#5065]
### Your use case

The actual wording of the persistent notification is:  "Listening for events" .
Some tchap users have told us that the word "listening" does not give enough context and can be scary.
Maybe we could use another word like: synchronize or recover ? 
Some propositions: "Element/Tchap synchronizes your messages", "Element/Tchap recovers your messages" ?

**Comment by bmarty** [comment]
@daniellekirkwood any thoughts on that point?

**Comment by daniellekirkwood** [comment]
I'm sure we recently updated this but I'm struggling to find the wording we used or GH issue. @ouchadam do you recall this discussion that we had?

**Comment by ouchadam** [comment]
yep it's here https://github.com/vector-im/element-android/issues/4298#issuecomment-952995120 originally this copy said `Listening for events` but we've changed it to `Listening for notifications` https://github.com/vector-im/element-android/pull/4314

This persistent notification is a requirement of the android OS in order to keep the non google play version of the app running in the background and able to receive messages/notifications/events

originally the notification would briefly show each time a sync occurred (every 60 seconds or so) but the time in between syncs allowed OEM battery optimisations to kick in and shutdown the app until the next manual launch

**Comment by bmarty** [comment]
Thanks. `notification_listening_for_events` is still used for the notification channel name, which is less visible to the user but still visible from the setting (and from the notification when long clicking on it):

<img width="479" alt="image" src="https://user-images.githubusercontent.com/3940906/151205643-3dbdc050-56bc-49c0-933e-d33c78c34faf.png">

Anyway the wording for the notification itself has been changed in Element Android v1.3.7 (2021-11-04) @ClaireGizard

**Comment by daniellekirkwood** [comment]
If we have updated the content here, can we close this issue? 

If no, what are we waiting for in order to move forwards?

**Comment by ouchadam** [comment]
I believe the intention of this issue was to see if we can avoid using `listening`, whereas the recent updates we've made were to swap `events` to `notifications`

**Comment by lrq3000** [comment]
I would suggest `Background notifications` or `Background events updates`? The option is specifically to setup how background events are updated, not when Element is on the foreground, so IMHO it would be much clearer to state it like this.
